### PR TITLE
Add prompt to switch network on jars of unconnected network

### LINKED ISSUE #272: Prompt user to switch networks in farm row body
The farm row body content (highlighted below) shouldn't show controls allowing the user to interact with the contract unless they are on the relevant network.

farmbody

Instead, tell them they need to switch to a network the farm pertains to.

### PR DESCRIPTION
**Pull request by larrythecucumber321**
closes #272 - only added prompt on first button.
image

### CODE REVIEW

File: v2/features/farms/FarmsTableRowBody.tsx [hunk 1]
```diff
@@ -10,11 +13,39 @@ import { CoreSelectors } from "v2/store/core";
...
 }
 
+const ConnectButton: FC<{ network: Network | undefined }> = ({ network }) => {
+  const { library } = useWeb3React<Web3Provider>();
+  const allCore = useSelector(CoreSelectors.selectCore);
+
+  if (!network || !allCore || !library) return null;
+  return (
+    <Button onClick={() => switchChain(library, network.chainId, allCore)}>
+      <span>Connect to</span>
```
Comment (janklimo) [inline v2/features/farms/FarmsTableRowBody.tsx:31]: @larrythecucumber321 can you please handle i18n here?

**Review by janklimo — COMMENTED** [review]
Love this feature @larrythecucumber321 My original idea was to replace the whole farm row body with a prompt to switch chains but I like this approach even more since it gives the user a better idea of what steps it unlocks 💯 just one little thing, you're welcome to merge after that

File: v2/features/farms/FarmsTableRowBody.tsx [hunk 2]
```diff
@@ -10,11 +13,39 @@ import { CoreSelectors } from "v2/store/core";
...
+      <div className="w-4 h-4 mr-1 ml-1">
+        <Image
+          src={`/networks/${network.name}.png`}
+          width={200}
+          height={200}
+          layout="responsive"
```
Comment (janklimo) [inline v2/features/farms/FarmsTableRowBody.tsx:37]: Please make this `intrinsic` for the smallest file size. That will make the image the size the wrapping div (it already has fixed width) and add 2x and 3x srcsets for higher DPI screens.

Comment (larrythecucumber321) [reply v2/features/farms/FarmsTableRowBody.tsx]: done!

File: v2/features/farms/FarmsTableRowBody.tsx (hunk 1 above)
Comment (larrythecucumber321) [reply v2/features/farms/FarmsTableRowBody.tsx]: done!

**Pull request merged**